main.tf

```diff
@@ -13,3 +18,17 @@ resource "aws_lambda_function" "default" {
   filename         = data.archive_file.selected.output_path
   source_code_hash = data.archive_file.selected.output_base64sha256
 }
+
+# CloudFront requires a version, and does not support the use of
+# $LATEST. Terraform's data source for aws_lambda_function returns
+# $LATEST for the qualified ARN. To work around this we create an
+# alias latest that points to the most recently published version
+# of the lambda code.
+#
+# https://github.com/terraform-providers/terraform-provider-aws/issues/10038
+resource "aws_lambda_alias" "default" {
+  name             = "latest"
+  description      = "The latest version"
```

Comment:
Prefer the wording "Latest published version" since it concisely says exactly what's going on.

Reply:
Done.